jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright 2016 University of Szeged.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+import sys
+
+PORT = 5001
+HOST = "localhost"
+
+def main():
+    try:
+        client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        client_socket.connect((HOST, PORT))
+    except socket.error, msg:
+        print('Failed to create the socket. Error code: %d, Error message: %s' % (msg[0], msg[1]))
+        sys.exit()
```

Comment:
Return code is missing from here. Probably use sys.exit(1) or you might use the error code above but not sure whether is this the common way to do this.

Hint: you can easily check the sys.exit() usage in the Jerry's python scripts with the following command:
find . -name "*.py" | xargs grep --color "sys.exit"